integration_tests/generics_array_01.py

```diff
@@ -0,0 +1,15 @@
+from ltypes import TypeVar
+
+T = TypeVar('T')
+
+def f(lst: T[:], i: T) -> T:
```

Comment:
A more interesting case would be to support `def f(lst: T[...], i: T) -> T`. We can do it later but we should keep in mind to support it.

Reply:
I haven't tried that out with CPython, but I bet the same issue as I mentioned above has to be solved first as well.